### PR TITLE
[Documentation] update student exam documentation with new screenshots and descriptions

### PR DESCRIPTION
**Pull request by PatZintel**
### Checklist
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.ase.in.tum.de/dev/guidelines/language-guidelines/).

### Steps for Testing
review the documentation

### Screenshots

Exercise Overview Part: 
image

New Programming Exercise Screenshot:
image

New Build indicators:
image

### CODE REVIEW

File: docs/user/exams/students_guide.rst [hunk 1]
```diff
@@ -117,7 +120,7 @@ Exam Conduction
 - You can also navigate through the exercises when you are done with one by clicking |save_continue|. This action will save and submit your changes and move to the next exercise.
 
     .. warning::
-        For programming exercises, you must manually press |submit| otherwise your solution will **not** be counted!
+        For programming exercises, there is no save button. You must manually press |submit| otherwise your solution will **not** be counted!
```
Comment (niclasschuemann) [inline docs/user/exams/students_guide.rst:123]:
```suggestion
        For programming exercises, there is no save button. You must manually press |submit| otherwise, your solution will **not** be graded!
```

File: docs/user/exams/students_guide.rst [hunk 2]
```diff
@@ -107,7 +107,10 @@ Welcome Screen
 
 Exam Conduction
 ^^^^^^^^^^^^^^^
-- Once the exam working time starts and you have confirmed your participation, the *Conduction* screen will appear.
+- Once the exam working time starts and you have confirmed your participation, the *Exercise Overview* screen will appear. This screen lists all exercises that are part of your exam with their respective amount of points, title and type of exercise. The status column indicates the status of each exercises and whether you have a submission in them or not.
```
Comment (niclasschuemann) [inline docs/user/exams/students_guide.rst:110]:
```suggestion
- Once the exam working time starts and you have confirmed your participation, the *Exercise Overview* screen will appear. This screen lists all exercises that are part of your exam with their respective amount of points, title and exercise type. The status column indicates the status of each exercise and whether you have a submission in them or not.
```

File: docs/user/exams/students_guide.rst [hunk 3]
```diff
@@ -217,9 +220,6 @@ Participating in Programming Exercises
     - You can add new files and directories using the |add_file|  and |add_folder| buttons.
     - You also have the ability to rename |rename| and delete |delete| files and folders, therefore **caution** is advised.
 
-        .. note::
-            If you accidentally delete or remove a file, you can use |refresh_files|, to load the last saved version from the server.
```
Comment (niclasschuemann) [inline docs/user/exams/students_guide.rst:221]: Why is this removed?

**Review by niclasschuemann — COMMENTED** [review]
I have left some suggestions, would be great if you could look at them :)

File: docs/user/exams/students_guide.rst (hunk 3 above)
Comment (PatZintel) [reply docs/user/exams/students_guide.rst:221]: This doesn't work anymore ^^' since the programming exercises now get autosaved immediately when a change happens, reloading the files after you deleted a file doesn't bring the file back anymore.

Comment (niclasschuemann) [reply docs/user/exams/students_guide.rst:221]: I just tested this and you are right - It just undoes all uncommitted changes in files that were not deleted. As long as this issue is not resolved, we should indeed delete this wrong information from the doc.
Thanks for the clarification!

**Review by niclasschuemann — APPROVED** [review]
Updated screenshots reflect the current system, good job!

**Review by niclasschuemann — DISMISSED** [review]

**Review by niclasschuemann — APPROVED** [review]
Re-approve
